### DISCUSSION TITLE
nextjs v15 and react v19 will this plugin support it?

### DISCUSSION THREAD

**Discussion by CreativeArtDesign** [vvo/iron-session#886] [Ideas]
Will this plugin be updated to support react v19

When i trying to install it, it compains on unsupported version of react 19..

**Comment by vvo** [comment]
Yes, v9 works with both. The examples in this repo run on Next 16.3 and React 19.2, and CI typechecks against the real Next.js types, so a Next release breaks our build instead of yours.

iron-session has no `react` or `next` dependency, not even a peer one, so the version complaint you saw came from something else in your tree.

v9 needs Node 22.13+ and is ESM-only: https://github.com/vvo/iron-session/releases/tag/v9.0.0
